x-pack/plugins/security/server/prompt_page.tsx

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
__note:__ the name of this page isn't super descriptive and based on the name of its main `EuiEmptyPrompt` component, happy to take any other suggestions if you feel `Prompt` is confusing.